src/components/Modal/ModalContent/MovieForm/MovieForm.jsx

```diff
@@ -1,78 +1,204 @@
+                        </FormRow>
+                        {/*<FormRow label="Genre">*/}
+                        {/*    <div className="select">*/}
+                        {/*        <FieldArray*/}
+                        {/*            as="select"*/}
+                        {/*            name="genres"*/}
+                        {/*            multiple={true}*/}
+                        {/*            onChange={handleChange}*/}
+                        {/*            onBlur={handleBlur}*/}
+                        {/*        >*/}
+                        {/*            {values.genres.map((option, index) => <option key={`option-${index}`} value={index}>{option}</option>)}*/}
+                        {/*        </FieldArray>*/}
+                        {/*    </div>*/}
+                        {/*</FormRow>*/}
+                        <FormRow label="Overview">
+                            <input
+                                id="Overview"
+                                type="text"
+                                name="overview"
+                                placeholder="Overview here"
+                                value={values.overview}
+                                onChange={handleChange}
+                                onBlur={handleBlur}
+                            />
+                        </FormRow>
+                        <FormRow label="Runtime">
+                            <input
+                                id="Runtime"
+                                type="text"
+                                name="runtime"
+                                placeholder="Runtime here"
+                                value={values.runtime}
+                                onChange={handleChange}
+                                onBlur={handleBlur}
+                            />
+                        </FormRow>
+
+                        <div className="buttons-holder">
+                            <Button type="button" className="btn-outline" title="reset" onButtonClick={() => {console.log("reset")}} />
+                            {props.isEditMovieForm ? (
+                                <Button title="save" type="button" onButtonClick={(e) => {submitForm(); e.preventDefault();}} />
+                            ): <Button title="submit" type="submit" onButtonClick={(e) => {submitForm(); e.preventDefault();}}/> }
+                        </div>
+                    </form>
+                )
+            }}
+        </Formik>
     )
 };
+
+const mapStateToProps = (state, ownProps) => {
+     if (ownProps.isEditMovieForm) {
+        const movieIndex = state.movieReducer.moviesList.findIndex(movie => movie.id === ownProps.movieId);
+
+        let currentMovie = null;
+        if (movieIndex !== -1) {
+            currentMovie = state.movieReducer.moviesList[movieIndex];
+        }
+
+        return {
+            movie: currentMovie
+        }
+     } else {
+         return {}
+     }
```

Comment:
from my point of view it's better to always return object with same interface even though some of it values are going to be `undefined` or `null`

Reply:
done